### PR TITLE
Se cambio el item del cliente

### PR DESCRIPTION
**Pull request by KarlaCMei**

### CODE REVIEW

**Comment by KarlaCMei** [comment]
[Item clients](https://trello.com/c/jaHPZsRC/61-item-clients)

File: app/src/main/java/com/epacheco/reports/view/clientView/clientView/AdapterClients.java [hunk 1]
```diff
@@ -86,14 +86,14 @@ public void onClick(View v) {
       }
     });
 
-    holderClients.btnDetail.setOnClickListener(new OnClickListener() {
+    /*holderClients.btnDetail.setOnClickListener(new OnClickListener() {
```
Comment (dalvik31) [inline app/src/main/java/com/epacheco/reports/view/clientView/clientView/AdapterClients.java:89]: Here, remove commented code

File: app/src/main/java/com/epacheco/reports/view/clientView/clientView/ClientsViewClass.java [hunk 2]
```diff
@@ -13,7 +13,10 @@
 import androidx.fragment.app.FragmentActivity;
 import androidx.appcompat.app.AppCompatActivity;
 import android.os.Bundle;
+
+import androidx.recyclerview.widget.GridLayoutManager;
```
Comment (dalvik31) [inline app/src/main/java/com/epacheco/reports/view/clientView/clientView/ClientsViewClass.java:17]: Here, remove imports if you are not using it

File: app/src/main/java/com/epacheco/reports/view/clientView/clientView/ClientsViewClass.java [hunk 3]
```diff
@@ -106,7 +109,8 @@ public void successDownloadClients(ArrayList<Client> listCliets) {
     hideProgress();
     if(listCliets!=null && !listCliets.isEmpty()){
       binding.recyclerListClient.setHasFixedSize(true);
-      binding.recyclerListClient.setLayoutManager(new LinearLayoutManager(this));
+      binding.recyclerListClient.setLayoutManager(new StaggeredGridLayoutManager(2,StaggeredGridLayoutManager.VERTICAL));
+      //binding.recyclerListClient.setLayoutManager(new GridLayoutManager(this, 2));
```
Comment (dalvik31) [inline app/src/main/java/com/epacheco/reports/view/clientView/clientView/ClientsViewClass.java:113]: Here, remove commented code

File: app/src/main/res/layout/item_new_client.xml [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        android:padding="10dp">
+
+
+        <TextView
+            android:id="@+id/lbl_client_name"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="5dp"
+            android:fontFamily="sans-serif-black"
+            android:text="Nombre"
```
Comment (dalvik31) [inline app/src/main/res/layout/item_new_client.xml:27]: Here, move this string to the correct file

**Pull request merged**